From: Weng Xuetian <wengxt@gmail.com>
Date: Sun, 25 Mar 2018 10:16:16 -0700
Subject: Fix #34
Applied-Upstream: commit:af033e3d5305108eecc568adff7f8b2da5831ed6

Add include <QIcon>

This fixes FTBFS with Qt 5.11.
---
 quickphrase-editor/batchdialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/quickphrase-editor/batchdialog.cpp b/quickphrase-editor/batchdialog.cpp
index 5b4c72c..4038db5 100644
--- a/quickphrase-editor/batchdialog.cpp
+++ b/quickphrase-editor/batchdialog.cpp
@@ -17,6 +17,7 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <QIcon>
 #include "batchdialog.h"
 #include "common.h"
 #include "ui_batchdialog.h"
